reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/add-infra-node.json

```diff
@@ -10,6 +10,9 @@
     "AmiId": {
       "Type": "AWS::EC2::Image::Id"
     },
+    "ExistingStack": {
```

Comment:
You could also use the [cloudformation_facts module](http://docs.ansible.com/ansible/cloudformation_facts_module.html) to validate the stack already exists. It could also help to provide defaults for the new stack based on the primary stack.